examples/bpk-component-section-header/examples.tsx

```diff
@@ -0,0 +1,85 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow strict */
+
+import BpkSectionHeader from '../../packages/bpk-component-section-header';
+import BpkButton from '../../packages/bpk-component-button';
+import SmallLongArrowRightIcon from '../../packages/bpk-component-icon/sm/long-arrow-right';
+import SmallLongArrowLeftIcon from '../../packages/bpk-component-icon/sm/long-arrow-left';
+import { withButtonAlignment } from '../../packages/bpk-component-icon/index';
+import { cssModules, isRTL } from '../../packages/bpk-react-utils';
+
+import STYLES from './examples.module.scss';
+
+const cls = cssModules(STYLES);
+
+const AlignedSmallLongArrowRightIcon = withButtonAlignment(
+  SmallLongArrowRightIcon,
+);
+
+const AlignedSmallLongArrowLeftIcon = withButtonAlignment(
+  SmallLongArrowLeftIcon,
+);
+
+const btnNode = <BpkButton CommonProps={{ blank: false }}>Action</BpkButton>;
+const btnWithOnlyIconNode = (
+  <BpkButton
+    className={cls('bpk-component-section-header--btn')}
+    CommonProps={{ blank: false }}
+  >
```

Comment:
What is this prop used for?

Reply:
if missing it, got this warning 
![image](https://github.com/Skyscanner/backpack/assets/139192353/a70d0f28-58ef-4807-a77f-75e1b2a4a036)

Reply:
should I remove CommonProps?

Reply:
Oh that's weird.. actually BpkButton is not written in TS. Instead you can use BpkButtonV2 which is the newer version and is written in TS as the old one will go away anyways 🙂
`import { BpkButtonV2 } from '../../packages/bpk-component-button';`

Reply:
CommonProps is not actually a prop, I guess the TS compiler gets a bit confused with importing a JS component into a TS file 🙁

Reply:
`BpkButtonV2` works, thank you